hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/scm/ReconStorageContainerManagerFacade.java

```diff
@@ -331,7 +339,10 @@ public void start() {
       initializePipelinesFromScm();
     }
     getDatanodeProtocolServer().start();
-    this.reconScmTasks.forEach(ReconScmTask::start);
+    reconSafeModeMgrTask.start();
```

Comment:
This is a blocking call for Recon Constructor, and no one will be able to communicate for the 5 minute. Even SCM sync is not enabled during this time.
Can this is refactored to avoid impacted services like health task or others, and to be non-blocking.

Reply:
Thanks @sumitagrawl for review, however as discussed, this is not blocking call.

Reply:
@devmadhuu Just to clarify, do you mean although `start()` blocks the current thread, other threads would still be able to handle Web UI and other RPC calls?